internal/pkg/describe/service.go

```diff
@@ -296,6 +300,32 @@ func (d *ecsServiceDescriber) ServiceConnectDNSNames() ([]string, error) {
 	return service.ServiceConnectAliases(), nil
 }
 
+// DeploymentConfigAlarmNames returns the rollback alarm names of a service.
+func (d *ecsServiceDescriber) DeploymentConfigAlarmNames() ([]string, error) {
+	service, err := d.ecsClient.Service(d.app, d.env, d.service)
+	if err != nil {
+		return nil, fmt.Errorf("get service %s: %w", d.service, err)
+	}
+	if service.DeploymentConfiguration.Alarms == nil {
+		return nil, nil
+	}
+	return aws.StringValueSlice(service.DeploymentConfiguration.Alarms.AlarmNames), nil
+}
+
+// DeploymentType returns 'Alarm-Based Rollback' or 'Default'.
+func (d *ecsServiceDescriber) DeploymentType() (string, error) {
+	deploymentType := "Default"
```

Comment:
I'm not wed to this term. How do people feel about "Rolling"? My one concern is that folks may not realize that is the default deployment type.